### PR TITLE
Add 3.5.0 release changelog

### PR DESCRIPTION
**Pull request by Didainius**
Add 3.5.0 release changelog

### CODE REVIEW

File: .changes/v3.5.0/733-features.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 * **New Resource:** `vcd_library_certificate` for managing certificates in library (provider configuration) [GH-733]
-* **New Data source:** `vcd_library_certificate` for reading certificates in library [GH-733]
\ No newline at end of file
+* **New Data source:** `vcd_library_certificate` for reading certificates in library [GH-733]
```
Comment (Didainius) [inline .changes/v3.5.0/733-features.md:2]: Added newline

File: .changes/v3.5.0/758-bug-fixes.md [hunk 2]
```diff
@@ -1 +1 @@
-* Fix bug where using `vcd_vm_internal_disk` could remove `description` field in `vcd_vapp_vm` and `vcd_vm` [GH-758]
\ No newline at end of file
+* Fix bug where using `vcd_vm_internal_disk` could remove `description` field in `vcd_vapp_vm` and `vcd_vm` [GH-758]
```
Comment (Didainius) [inline .changes/v3.5.0/758-bug-fixes.md:1]: Added newline

File: .changes/v3.5.0/751-improvements.md [hunk 3]
```diff
@@ -1 +1 @@
-* Bump Terraform SDK to 2.10.0 [GH-751]
\ No newline at end of file
+* Bump Terraform SDK to 2.10.0 [GH-751]
```
Comment (Didainius) [inline .changes/v3.5.0/751-improvements.md:1]: Added newline

**Review by vbauzysvmware — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,44 @@
...
+* **New Data source:** `vcd_nsxt_alb_settings` for reading NSX-T ALB on NSX-T Edge Gateways  ([#726](https://github.com/vmware/terraform-provider-vcd/pull/726))
+* **New Resource:** `vcd_library_certificate` for managing certificates in library (provider configuration) ([#733](https://github.com/vmware/terraform-provider-vcd/pull/733))
+* **New Data source:** `vcd_library_certificate` for reading certificates in library ([#733](https://github.com/vmware/terraform-provider-vcd/pull/733))
+* **New Resource:** `vcd_nsxt_alb_edgegateway_service_engine_group` for managing NSX-T ALB Service Engine Groups
+  assignments to Edge Gateways ([#738](https://github.com/vmware/terraform-provider-vcd/pull/738), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Data source:** `vcd_nsxt_alb_edgegateway_service_engine_group` for reading NSX-T ALB Service Engine Groups
+  assignments to Edge Gateways ([#738](https://github.com/vmware/terraform-provider-vcd/pull/738), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Resource:** `vcd_vdc_group` for managing VDC groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
```
Comment (lvirbalas) [inline CHANGELOG.md:12]:
```suggestion
* **New Resource:** `vcd_vdc_group` for managing VDC Groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,44 @@
...
+* **New Data source:** `vcd_vdc_group` for reading VDC groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
```
Comment (lvirbalas) [inline CHANGELOG.md:13]:
```suggestion
* **New Data source:** `vcd_vdc_group` for reading VDC Groups ([#752](https://github.com/vmware/terraform-provider-vcd/pull/752))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,44 @@
...
+* **New Resource:** `vcd_nsxt_alb_pool` for NSX-T Load balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
```
Comment (lvirbalas) [inline CHANGELOG.md:14]:
```suggestion
* **New Resource:** `vcd_nsxt_alb_pool` for NSX-T Load Balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,44 @@
...
+* **New Data source:** `vcd_nsxt_alb_pool` for reading NSX-T Load balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
```
Comment (lvirbalas) [inline CHANGELOG.md:15]:
```suggestion
* **New Data source:** `vcd_nsxt_alb_pool` for reading NSX-T Load Balancer pools ([#756](https://github.com/vmware/terraform-provider-vcd/pull/756))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,44 @@
...
+* **New Resource:** `vcd_nsxt_alb_virtual_service` for managing NSX-T ALB Virtual Service on NSX-T Edge Gateways
+  ([#757](https://github.com/vmware/terraform-provider-vcd/pull/757), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+* **New Data source:** `vcd_nsxt_alb_virtual_service` for reading NSX-T ALB Virtual Service on NSX-T Edge Gateways
+  ([#757](https://github.com/vmware/terraform-provider-vcd/pull/757), [#764](https://github.com/vmware/terraform-provider-vcd/pull/764))
+
+## IMPROVEMENTS
+* Remove Coverity warnings from code ([#734](https://github.com/vmware/terraform-provider-vcd/pull/734))
+* Add support for disk controller type nvme ([#680](https://github.com/vmware/terraform-provider-vcd/pull/680),[#739](https://github.com/vmware/terraform-provider-vcd/pull/739))
```
Comment (lvirbalas) [inline CHANGELOG.md:23]: I think we should have the corresponding resource name at the beginning of the sentence (like a few lines below).

**Review by lvirbalas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (Didainius) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 5 above)
Comment (Didainius) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 6 above)
Comment (Didainius) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 7 above)
Comment (Didainius) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 8 above)
Comment (Didainius) [reply CHANGELOG.md]: Added.

**Review by lvirbalas — APPROVED** [review]
Thanks for the tune-ups!

**Pull request merged**